networkd/network_monitor_thread.ml

```diff
@@ -229,7 +229,7 @@ let rec monitor dbg () =
 			) devs
 		in
 		
-		let bonds : (string * string list) list = Network_server.Bridge.get_all_bonds () dbg ~from_cache:true () in
+		let bonds : (string * string list) list = Network_server.Bridge.get_all_bonds dbg true in
```

Comment:
Please keep the label if it does not confuse the compiler. That `true` on its own is confusing, I'd rather do
`let from_cache = true in ...` if we cannot keep the labels

Reply:
Sadly we cannot keep the labels, so we should add the `let` statement